requester/print.go

```diff
@@ -32,8 +32,15 @@ type report struct {
 	average  float64
 	rps      float64
 
+	avgConn  float64
+	avgDns   float64
+	avgReq   float64
+	avgRes   float64
+	avgDelay float64
+
 	results chan *result
 	total   time.Duration
+	trace   bool
```

Comment:
move this right before avgConn bool and comment that if it is set, the following fields will be filled.